source/developers-guide/rest-api/api-resource-manufacturers/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: REST API - Manufacturers Resource
+github_link: developers-guide/rest-api/api-resource-manufacturers/index.md
+shopware_version: 5.2.0
```

Comment:
I'll change the ticket to SW-13681 which includes these changes for the mink test.